### PR TITLE
speedcrunch: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** SpeedCrunch 0.12

### CODE REVIEW

File: pages/common/speedcrunch.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Copy the result of the most recent calculation:
+
+`Ctrl + R`
+
+- Open the formula book:
+
+`Ctrl + 1`
+
+- Clear calculator of calculations:
```
Comment (CleanMachine1) [inline pages/common/speedcrunch.md:18]:
```suggestion
- Clear the calculator of recent calculations:
```

File: pages/common/speedcrunch.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`Ctrl + N`
+
+- Wrap highlighted in parathesis (defaults to wrapping all if nothing selected):
```
Comment (CleanMachine1) [inline pages/common/speedcrunch.md:22]:
```suggestion
- Wrap highlighted in parenthesizes (defaults to wrapping all if nothing selected):
```

**Review by EmilySeville7cfg — APPROVED** [review]
Everything is fine. 😄

File: pages/common/speedcrunch.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# speedcrunch
+
+> A high-precision scientific calculator.
+> More information: <https://www.speedcrunch.org/>.
```
Comment (navarroaxel) [inline pages/common/speedcrunch.md:4]:
```suggestion
> More information: <https://www.speedcrunch.org>.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**